packages/react-native-editor/package.json

```diff
@@ -100,7 +100,7 @@
 		"ios": "react-native run-ios",
 		"ios:fast": "react-native run-ios",
 		"test": "cross-env NODE_ENV=test jest --verbose --config ../../test/native/jest.config.js",
-		"test:debug": "cross-env NODE_ENV=test node --inspect-brk jest --runInBand --verbose --config ../../test/native/jest.config.js",
+		"test:debug": "cross-env NODE_ENV=test node --inspect-brk ../../node_modules/.bin/jest --runInBand --verbose --config ../../test/native/jest.config.js",
```

Comment:
The context for this change is that `npm run native test:debug` has been broken (possibly since the RN 0.64 upgrade) with an error where it can't find `jest`:
```
VM50 loader.js:902 Uncaught Error: Cannot find module '/Users/me/projects/gutenberg/packages/react-native-editor/jest'
    at Function.Module._resolveFilename (VM50 loader.js:902)
    at Function.Module._load (VM50 loader.js:746)
    at Function.executeUserEntryPoint [as runMain] (VM74 run_main.js:76)
    at VM45 run_main_module.js:17
```
Updating the path to what the (now deleted) `npm run test-unit:native:debug` command [used](https://github.com/WordPress/gutenberg/pull/33657/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L279) fixes the error.